Multisig/UI/Settings/SettingsTopTabView.swift

```diff
@@ -10,33 +10,28 @@ import SwiftUI
 
 struct SettingsTopTabView: View {
```

Comment:
I would remove the word "Top" wherever it occurs

Reply:
I will rename it to `SettingsContainerView`

Reply:
Good one, also please remove "Top" from the other areas